prisma/models/ProductToTag.prisma

```diff
@@ -0,0 +1,6 @@
+model ProductToTag {
+  product_id           String  @unique
+  product              Product @relation("product_tag", fields: [product_id], references: [id], onDelete: Cascade)
+  tag_name             String  @unique
+  tag                  Tag @relation("tag_product", fields: [tag_id], references: [id], onDelete: Cascade)
```

Comment:
I think you might have meant to use `tag_name` instead of `tag_id` in your relation (or the other way around).

Reply:
Actually, after looking around more I realized you were probably going for referencing by name rather than ID, so you can just change `tag_id` to `tag_name` and `id` to `name`.